composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "license": "GPL-3.0+",
   "require": {
+    "ext-intl": "*",
```

Comment:
Good idea! And we can make it simpler even, because the Twig defaults are the same as you've set there (dot and comma).